api/app_analytics/influxdb_wrapper.py

```diff
@@ -90,7 +91,7 @@ def get_events_for_organisation(organisation_id: id, date_range: str = "30d"):
         filters=f'|> filter(fn:(r) => r._measurement == "api_call") \
         |> filter(fn: (r) => r["_field"] == "request_count") \
         |> filter(fn: (r) => r["organisation_id"] == "{organisation_id}")',
-        drop_columns='"organisation", "project", "project_id"',
+        drop_columns='"organisation", "project", "project_id", "environment", "environment_id"',
```

Comment:
Why is the list of columns different here? Would it matter if we defined all of them here too and just used a single constant? `DEFAULT_DROP_COLUMNS` for example? Can we make it a list too rather than a string?

Reply:
OK sure can you do that!